### ISSUE TITLE
MediaIoBaseDownload doesn't restore httplib2 follow_redirects state after use.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#280]
```
MediaIoBaseDownload doesn't restore httplib2 follow_redirects state after use.

```

Original issue reported on code.google.com by `jcgregorio@google.com` on 13 Jun 2013 at 3:05

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 6a9f2cfb7cba.
```

Original comment by `jcgregorio@google.com` on 13 Jun 2013 at 6:48
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 1ab344e0a34d.
```

Original comment by `jcgregorio@google.com` on 19 Jun 2013 at 5:25
